w03h03/test/pgdp/SeamCarvingTest.java

```diff
@@ -158,20 +158,43 @@ void testing_build_seams_seamsArray(int[][] seams, long[] seamWeights, int[] gra
 
         static Stream<Arguments> testing_build_seams_seamsArray() {
             return Stream.of(
-                    arguments(
-                            new int[6][5],
-                            new long[6],
-                            new int[]{
-                                    Integer.MAX_VALUE, Integer.MAX_VALUE, Integer.MAX_VALUE, Integer.MAX_VALUE, Integer.MAX_VALUE, Integer.MAX_VALUE,
-                                    1, 1, 1, 1, 1, 1,
-                                    69, 420, 42, 42, 42, 1337,
-                                    100, 10, 666, 20, 3, 10,
-                                    Integer.MAX_VALUE, Integer.MAX_VALUE, Integer.MAX_VALUE, Integer.MAX_VALUE, Integer.MAX_VALUE, Integer.MAX_VALUE
-                            },
-                            6,
-                            5,
-                            new int[][]{new int[]{0, 0, 0, 1, 1}, new int[]{1, 1, 2, 1, 1}, new int[]{2, 2, 2, 1, 1}, new int[]{3, 3, 3, 4, 4}, new int[]{4, 4, 4, 4, 4}, new int[]{5, 5, 4, 4, 4}}
-                    )
+                arguments(
+                    new int[6][5],
+                    new long[6],
+                    new int[] {
+                        Integer.MAX_VALUE, Integer.MAX_VALUE, Integer.MAX_VALUE, Integer.MAX_VALUE, Integer.MAX_VALUE,
```

Comment:
please reformat so every line has width elements (it was properly formatted before)

Reply:
> please reformat so every line has width elements (it was properly formatted before)

I don't exactly get what you mean by 'width elements'. Indentation or the array initialization? Anyways, feel free to format it as you like. 😄